src/perform_step/verner_rk_perform_step.jl

```diff
@@ -675,43 +675,42 @@ end
     atmp = calculate_residuals(utilde, uprev, u, integrator.opts.abstol, integrator.opts.reltol,integrator.opts.internalnorm)
     integrator.EEst = integrator.opts.internalnorm(atmp)
   end
-  integrator.k[1]=k1; integrator.k[2]=k2;
-  integrator.k[3]=k3; integrator.k[4]=k4;
-  integrator.k[5]=k5; integrator.k[6]=k6;
-  integrator.k[7]=k7; integrator.k[8]=k8;
-  integrator.k[9]=k9; integrator.k[10]=k10;
-  integrator.k[11]=k11; integrator.k[12]=k12;
-  integrator.k[13]=k13; integrator.k[14]=k14;
-  integrator.k[15]=k15; integrator.k[16]=k16
+  integrator.k[1]=k1; integrator.k[2]=k8;
+  integrator.k[3]=k9; integrator.k[4]=k10;
+  integrator.k[5]=k11; integrator.k[6]=k12;
+  integrator.k[7]=k13; integrator.k[8]=k14;
+  integrator.k[9]=k15; integrator.k[10]=k16
```

Comment:
Maybe add the same comment as below here as well?

Reply:
Yes, it's probably a good idea.

Reply:
Okay sure I will make the edit.